src/main/java/picard/fingerprint/CheckFingerprint.java

```diff
@@ -187,7 +189,7 @@
 
     @Argument(shortName = "G", doc = "File of genotypes (VCF) to be used in comparison. May contain " +
             "any number of genotypes; CheckFingerprint will use only those that are usable for fingerprinting.")
-    public File GENOTYPES;
+    public String GENOTYPES;
```

Comment:
Why not a URL?

Reply:
why yes? what's the advantage?

Reply:
I'll respond with a question: what can be represented as a String but not a Path/URL?

Reply:
The problem is that Path support for actual URLs (i.e. http://www.google.com is flaky at best...so calling the type a URL can cause confusion. I'm super-unhappy about using String and would love to come up with an alternative.

Reply:
I think this is such a slippery and steep slope I'd like others (who have more engineering background) to weigh in on this. ["With the first link, the chain is forged, ... <the first `File` to `String` conversion> ... chains us all, irrevocably"](http://memory-alpha.wikia.com/wiki/Aaron_Satie)

Reply:
The ideal would be to parse directly to `Path`, but one downside we've hit in other projects is that `Paths.get` (etc) uses whatever default storage options are set at the time, so if you want to tweak anything in the options you have to do so before parsing. We've been using `URI` at the parsing level to work around it.